src/include/daos_srv/container.h

```diff
@@ -87,7 +89,9 @@ struct ds_cont_child {
 	 * snapshot_delete_hlc(or rebuild), then aggregation needs to restart
 	 * from 0.
 	 */
-	uint64_t		sc_aggregation_full_scan_hlc;
+	uint64_t		sc_aggregation_vos_full_scan_hlc;
+	uint64_t		sc_aggregation_ec_full_scan_hlc;
```

Comment:
Could you remove these two variables from ds_cont_child? They are per-ULT, and you've already have per-ULT 'full_scan_hlc', so they looks useless to me now.

Reply:
oh, we can only fix this when we simplify cont_child_aggregate, i.e. turn it into get_epoch_list(), otherwise you have to maintain two full scan hlc for each ULT for sharing the current cont_child_aggregate.